EIPS/eip-4844.md

```diff
@@ -322,6 +322,8 @@ The actual `data_fee` as calculated via `calc_data_fee` is deducted from the sen
 
 ### Networking
 
+Blob transactions are not automatically broadcast to peers. Instead, they are only announced using `NewPooledTransactionHashes` messages, and can then be manually requested via `GetPooledTransactions`.
```

Comment:
```suggestion
Nodes MAY not automatically broadcast Blob transactions to peers. Instead, they are only announced using `NewPooledTransactionHashes` messages, and can then be manually requested via `GetPooledTransactions`.
```

[Editorial] I wouldn't be surprised if some nodes want to choose to broadcast such Blob transactions. Hence, I love to prompt author to clarify if the spec is  mandating(`MUST`) / suggesting(`SHOULD`) /or allowing (`MAY`) this implementation behavior

Reply:
This is a good point. The intention here is to explicitly mandate this behavior (so `MUST`), because that allows nodes to drop misbehaving peers.

Reply:
Cool. thanks for clarifying. In that case I'd suggest update to 

```suggestion
Nodes MUST not automatically broadcast Blob transactions to peers. Instead, they are only announced using `NewPooledTransactionHashes` messages, and can then be manually requested via `GetPooledTransactions`.

Any nodes received Blob transactions MUST deemed the sending nodes "misbehaving" and MUST drop them as peer.
```